packages/epo-widget-lib/src/widgets/FilterTool/FilterTool.tsx

```diff
@@ -48,16 +48,11 @@ const FilterTool: FunctionComponent<FilterToolProps> = ({
     }
   }, [selectedColor]);
 
-  const handleSelect = (event: FormEvent<HTMLSelectElement>) => {
-    const { target } = event;
-    const { value } = target as HTMLSelectElement;
-
-    return setSelectedColor(value as FilterColor);
-  };
-
   const isArrowHidden = (color: FilterColor) =>
     selectedColor !== color && selectedColor !== "none";
 
+  const selectLabel = t("filterTool.selectLabel");
```

Comment:
have to do this because `t()` returns `string | null` which is incompatible with the type for placeholder `string | undefined`, this is a known issue supposed to be resolved in a later version of i18next but for now assigning it to a new variable types it as `string`